ci/repokitteh/modules/ownerscheck.star

```diff
@@ -204,9 +204,9 @@ def _comment(config, results, assignees, sender, force=False):
         if assignee in members:
           api_assignee = assignee
           break
-      # Otherwise, pick at "random" (we just use timestamp).
+      # Otherwise, pick at random.
       if not api_assignee:
-        api_assignee = members[now().second % len(members)]
+        api_assignee = members[random.randint(1000000) % len(members)]
```

Comment:
@itayd to confirm, we don't need to do any explicit seed of the RNG and by default it gives us some reasonable entropy (at least for LB purposes)?

Reply:
Hmm, doesn't seem very random, seems to be assigning a fixed reviewer. Should I be seeding with something like time-of-day?

Reply:
that is odd. it should be random. maybe members have a single item? :) can you print the result of randint before using it?

Reply:
Turns out I just needed to roll the dice more, thanks!